components/locale-chooser.js

```diff
@@ -57,3 +130,40 @@ function getLocaleAndHost() {
     return {locale: fallbackLocale, host}
   }
 }
+
+function mapLocale(key) {
+  const svgStyle = {width: '1em', height: '100%'}
+  switch (key) {
+    case 'en':
+      return {
+        key,
+        display: 'English',
+        flag: <EnSvg {...svgStyle} />,
+      }
+    case 'es':
+      return {
+        key,
+        display: 'Español',
+        flag: <EsSvg {...svgStyle} />,
+      }
+    case 'fr': {
+      return {
+        key,
+        display: 'Français',
+        flag: <FrSvg {...svgStyle} />,
```

Comment:
Hmmm... I must be missing something.
When I'm just using the require I'm getting the error `Module parse failed: .\svgs\en.svg Unexpected token (1:0) You may need an appropriate loader to handle this file type.`
And with using `svg-react-loader` , I'm getting the error `Error: Cannot find module './svgs/en.svg'`.

I'll try again with a clear head tomorrow. Or if someone knows how to do it, help it welcome. 
It must be something really stupid tho :smile: